pyintegration/test_helper/__init__.py

```diff
@@ -61,7 +63,13 @@ def start_jvm():
             'workspace': os.path.realpath(workspace),
 
         }
-        jvm_options= {
+
+        if jvm_props:
+            jvm_properties.update(jvm_props)
+
+        print(jvm_properties)
```

Comment:
Was this on purpose, or meant to be debugging only?

Reply:
good catch again, I really should have taken a break and not been rushing things.